targets/kodi

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then
+    if release -le wheezy; then
+        error 99 "Kodi not supported on Debian Wheezy or older."
+    fi
+    if release = precise; then
+        error 99 "Kodi not supported on Precise Ubuntu."
+    fi
+fi
+REQUIRES='x11'
+PROVIDES='xbmc'
+DESCRIPTION='Installs the KODI media player. Ubuntu Only. (Approx. 140MB)'
+HOSTBIN='startkodi'
+CHROOTETC='kodi-keyboard.xml kodi-cycle.py'
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+
+## Debian Overrides
+# Kodi wiki recommends using the Jessie Backports repository
+if release = jessie; then
+    cat > '/etc/apt/sources.list.d/kodi.list' <<EOF
+# Bring in Jessie Backports for newer Kodi
+deb http://http.debian.net/debian jessie-backports main
+EOF
+    # Update database
+    apt-get update || true
+fi
+
+# Get rid of disruptive xserver-xorg-legacy package
+if release = stretch || release = sid || release = kali-rolling; then
+    remove xserver-xorg-legacy
```

Comment:
I put this workaround in so it would work when testing.  I think a dummy package is probably the way to go.  I'll remove this workaround from the kodi pull request now.  I will try to take a crack at the dummy package later.